homeassistant/components/mill/sensor.py

```diff
@@ -71,7 +70,7 @@ async def async_update(self):
             self._attr_state = _state
             return
 
-        if self.state not in [STATE_UNKNOWN, None] and _state < self.state:
+        if self.state is not None and _state < self.state:
```

Comment:
```suggestion
        if self._attr_state is not None and _state < self._attr_state:
```
Maybe? it's the only place in this file where `self.state` is used.

Reply:
Hmm, `self._attr_state` is used where we set the state. My understanding is that we should use `self.state` to read the state ?

Reply:
Might be right, just looked strange 😅